Docs: Clarify segmentMetadata cardinality, minmax, and size behavior. (#11549)
apache/druid: docs/querying/segmentmetadataquery.md

### cardinality

* `cardinality` in the result will return the number of unique values present in a string column. It is null for other column types.

>  If `merge` is set, the result will be the max of this value across segments. Only relevant for string columns.

Reviewer: This is not clear to us newbies. Does "max" mean the largest number of any segment, or the aggregated total across segments? Both are useful: if I have 1M rows, and see a cardinality of 1K, that could mean 1K total, or 1K per segment, which says something else if each segment has 1K rows...

Author: I pushed some new content that hopefully clears this up. Please let me know if it makes sense to you.

### minmax

* Estimated min/max values for each column. Only reported for string columns.

### size

> * `size` in the result will contain the estimated total segment byte size as if the data were stored in text format. This is _not_ the actual storage size of the column in Druid.

Reviewer: Pointer to where I might find the actual storage size? I want to know the amount of space the column takes so I know if it is worth the cost to store.

Author: I pushed some new content that hopefully clears this up. Please let me know if it makes sense to you.

### cardinality

* `cardinality` is the number of unique values present in string columns. It is null for other column types.

> This value is computed by examining the size of string column dictionaries. There is one dictionary per column per

Reviewer 2: ```suggestion
Druid examines the size of string column dictionaries to compute the cardinality value. There is one dictionary per column per
```
